projects/noloan/app/src/main/java/hasadna/noloan/SmsRecyclerAdapter.java

```diff
@@ -78,6 +79,18 @@ public void bind(SmsMessage sms) {
       Calendar calendar = Calendar.getInstance();
       Date receivedDate;
 
+      // Search if message was suggested by others -> update counter
+      if (DbMessages.getInstance().searchSuggestions(sms.getBody(), sms.getSender()) != null) {
```

Comment:
How about we pass the whole message to `searchSuggestions()` and inside it uses `body` and `sender`?

Reply:
Changed to that...